xml/html_title.xml

```diff
@@ -1597,6 +1605,14 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:manageengine:opmanager:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^ManageEngine Desktop Central 9$">
+    <description>ManageEngine Desktop Central 9</description>
+    <example>ManageEngine Desktop Central 9</example>
+    <param pos="0" name="service.vendor" value="ManageEngine"/>
+    <param pos="0" name="service.product" value="Desktop Central"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:manageengine:desktopcentral:-"/>
```

Comment:
This CPE is invalid. Was it autogenerated or manually created?

Reply:
Just a note @jvoisin - `update_cpes.py` removes *all* CPE entries and generates only entries which are valid according to the NIST database. It does so using the `vendor` and `product` fields and uses `cpe-remap.yaml` to try to find valid values when those fields are set to something that doesn't map and where we cannot change them.